### PR TITLE
review and update docs changed in #245

### PR DESCRIPTION
**Pull request by qiancai**
Signed-off-by: qiancai <qqzczy@126.com>

### What is changed?

In this PR, I've reviewed and updated the changes made in https://github.com/tikv/website/pull/245 from the writing perspective.

### CODE REVIEW

File: content/docs/5.1/develop/develop.md [hunk 1]
```diff
@@ -10,17 +10,17 @@ Learn how to use TiKV Clients for different languages.
...
-TiKV has clients for a number of languages:
+TiKV provides the following clients developed in different programming languages:
 
 - [Java Client](../clients/java)'s RawKV API is ready for production.
-- [Go Client](../clients/go) is still in the stage of prove-of-concept and under heavy development. 
-- [Rust Client](../clients/rust) is still in the stage of prove-of-concept and under heavy development.
-- [Python Client](../clients/python) is still in the stage of prove-of-concept and under heavy development.
-- [C++ Client](../clients/cpp) is still in the stage of prove-of-concept and under heavy development.
+- [Go Client](../clients/go) is still in the proof-of-concept stage and under development.
+- [Rust Client](../clients/rust) is still in the proof-of-concept stage and under development.
```
Comment (qiancai) [inline content/docs/5.1/develop/develop.md:17]: https://tikv.org/docs/5.1/develop/clients/rust/ 从这里看起来 Rust Client 是可用的呢。

File: content/docs/5.1/concepts/overview.md [hunk 2]
```diff
@@ -6,32 +6,32 @@ menu:
...
 ## Architecture
 
 A TiKV cluster consists of the following components:
 
-- A group of TiKV nodes: storing key-value pair data
-- A Placement Driver (PD) node: working as the manager of the TiKV cluster
+- A group of TiKV nodes: store key-value pair data
+- A group of Placement Driver (PD) nodes: work as the manager of the TiKV cluster
 
 TiKV clients interact with PD and TiKV through gRPC.
```
Comment (qiancai) [inline content/docs/5.1/concepts/overview.md:26]: 这里是否需要解释下 TiKV clients 是啥，就只是用来作为 PD 和 TiKV 之间交互的么？

Comment (marsishandsome) [reply content/docs/5.1/concepts/overview.md:26]: Clients 有自己隐含的意思，表示客户通过 clients 可以跟服务器进行交互。

File: content/docs/5.1/develop/develop.md (hunk 1 above)
Comment (marsishandsome) [reply content/docs/5.1/develop/develop.md:17]: 我加一下说明

**Review by marsishandsome — APPROVED** [review]

**Pull request merged**